### ISSUE TITLE
astropy build failing in travis-ci

### ISSUE THREAD

**Issue by wtgee** [astropy/astropy#5202]
This has been failing on my travis-ci builds for a number of days now no matter what I do. Any help would be appreciated. I posted about it in #5194 

```
astroplan/astroplan/conftest.py:53: in pytest_configure
    astropy_pytest_plugins.pytest_configure(config)
../../../miniconda/envs/test-environment/lib/python3.5/site-packages/astropy-1.2.1-py3.5-linux-x86_64.egg/astropy/tests/pytest_plugins.py:137: in pytest_configure
    if not config.getoption('remote_data'):
../../../miniconda/envs/test-environment/lib/python3.5/site-packages/_pytest/config.py:1061: in getoption
    raise ValueError("no option named %r" % (name,))
E   ValueError: no option named 'remote_data'
=========================== 1 error in 0.53 seconds ============================
```

**Comment by bsipocz** [comment]
@wtgee - Could you please copy over the full test run into a gist or link the travis builds? I have a few ideas, but they are guesswork at best with this little information.

**Comment by wtgee** [comment]
Thanks for looking into it! Let me know if I can help (or if it's just something I have set up wrong)

https://travis-ci.org/panoptes/POCS/jobs/147969377

**Comment by bsipocz** [comment]
@wtgee - And does the tests work if you run them locally from a clean conda environment? I didn't manage to get it installed and run, all sort of config errors are coming up, and not even reaching the point where travis fails.

**Comment by pllim** [comment]
I look at https://github.com/panoptes/POCS and I don't see the package template being used. And there is no `conftest.py`. Has it ever been successful before?

http://pytest.org/2.2.4/plugins.html

**Comment by wtgee** [comment]
The tests pass locally without any problems but it could easily be something on my end however the errors in travis are not very illustrative. From my perspective the tests are failing on the `remote_data` check in the line I linked earlier. Not sure why I would need to go through extra steps to make that work but my local environment could be tainted in some way. I'll check again from a clean environment.

@pllim what plugin do I need a conftest.py for?

**Comment by bsipocz** [comment]
@wtgee - Please try to run from a clean env. As I mentioned above I run into other config problems, so it seemed that it's a problem independent from astropy.

**Comment by wtgee** [comment]
100% clean environment (new instance of Debian 8.5 on GCE) works perfectly fine. Config problems are related to directory setup structure that is all specified in our `.travis.yml`. (Basically just creating a directory and setting an env variable)

I followed the exact steps that are outlined in the .travis.yml file and am able to run the `coverage run --source=pocs setup.py test` without problems. I cannot recreate the errors. Still willing to believe it's something I have set up wrong but it's only happening on travis.

https://github.com/panoptes/POCS/blob/develop/.travis.yml
